### PR TITLE
SRM release notes for 790

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/service-reliability-management.md [hunk 1]
```diff
@@ -13,6 +13,26 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## April xx, 2023, version 790xx
+
+### What's new
+
+Filters applied to the monitored services list on the Monitored Services page will get reset when you switch to a different project. (SRM-14383)
```
Comment (SmritiSatya) [inline release-notes/service-reliability-management.md:20]: Filters applied to the monitored services list on the **Monitored Services** page are reset when you switch to a different project.

File: release-notes/whats-new.md [hunk 2]
```diff
@@ -16,6 +16,14 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+
+## April xx, 2023, version 790xx
+
+### Service Reliability Management
+
+* Filters applied to the monitored services list on the Monitored Services page will get reset when you switch to a different project. (SRM-14383)
```
Comment (SmritiSatya) [inline release-notes/whats-new.md:24]: Filters applied to the monitored services list on the **Monitored Services** page are reset when you switch to a different project.

**Review by SmritiSatya — DISMISSED** [review]

File: release-notes/service-reliability-management.md (hunk 1 above)
Comment (SudheendraKatte) [reply release-notes/service-reliability-management.md:20]: Updated

**Comment by SudheendraKatte** [comment]
By-passing review steps as the change made is adding the release designation.

**Pull request merged**